### ISSUE TITLE
GL_ARB_shading_language_include not supported

### ISSUE THREAD

**Issue by karyon** [AtomLinter/linter-glsl#28]
this code

```
#extension GL_ARB_shading_language_include : require

#include </foo.bar.glsl>
```

triggers these errors:
```
glsl ERROR '#extension' : extension not supported: GL_ARB_shading_language_include at line 2 col 1
glsl ERROR '#include' : required extension not requested: GL_GOOGLE_include_directive at line 4 col 1
```
which seems a bit weird to me.

i suspect this is actually a problem of the validator but thought i should rather start here. let me know if i should report that to the validator project.

**Comment by Arcanemagus** [comment]
If you run `flslangValidator` manually on the file do you see the same messages? If so then it is certainly an issue that should be filed there instead of here.

If not, then we need to dig into how this package is configuring the linter to see if it's a misconfiguration there or not.

**Comment by karyon** [comment]
yeah i saw the same message. i had a look at the validator's code and it seems like it's intended this way, opened https://github.com/KhronosGroup/glslang/issues/249 nevertheless.
